capstone/src/main/java/com/google/sps/servlets/FileUnzipServlet.java

```diff
@@ -0,0 +1,161 @@
+package com.google.servlets;
+
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.BlobInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.ByteArrayInputStream;
+
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import java.util.Arrays;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import com.google.api.gax.paging.Page;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.MultipartConfig;
+
+
+@WebServlet("/unzip")
+@MultipartConfig
+public class FileUnzipServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+    
+    // String userId = request.getParamter("userId");
+    //Hard-coded UserId
+    String userId = "abcde";
+    
+    // The ID of GCP project
+    String projectId = "step-2020-team-2";
+
+    // The ID of GCS bucket
+    String bucketName = "vaderker-uploadedstoragebucket";
+
+    // String fileName = request.getParamter("files");
```

Comment:
same as above re TODO/etc

Reply:
TODO (github issue ref): To be replaced once we are no longer hardcoding userId.

Reply:
You need to add a TODO here as well.